### DISCUSSION TITLE
2.4 RC1 to RC2 update soup not updating?

### DISCUSSION THREAD

**Discussion by Phillip-Samson** [Security-Onion-Solutions/securityonion#10972] [2.4]
2.4 RC1 fresh install on Friday 8/5/2023
Distributed with only manager search node 
Oracle Linux
tried update to RC2 and it keeps trying to update soup it it normal to have to do 50 or more updates for soup

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36277967/c8e06b56-8e92-41aa-80da-85f78d41b739)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36277967/2d92e038-7ca1-4d9a-b462-3038a0ce7767)

**Comment by dougburks** [comment]
RC2 has not been officially announced yet. Please run the following:
```
sudo rm -f /tmp/supersoup/soup*
```
and then wait until RC2 has been announced.

**Reply by dougburks** [reply]
RC2 has officially been announced:
https://blog.securityonion.net/2023/08/security-onion-24-release-candidate-2.html
